### PR TITLE
Pages documentation

### PR DESCRIPTION
**Pull request by saachipahwa**

### CODE REVIEW

**Comment by saachipahwa** [comment]
Seems that new snack bar implementation has been added to flutter stable so i'll update our code and re-request review

**Comment by thevirtuoso1973** [comment]
I've already created an issue and PR for it, but you may take it from me if you wish. I'll do it tomorrow otherwise.

**Review by thevirtuoso1973 — COMMENTED** [review]
Resolve conflicts and re-request review.

File: lib/shared/friend_graph.dart [hunk 1]
```diff
@@ -3,6 +3,8 @@ import 'package:charts_flutter/flutter.dart' as charts;
 
 import 'package:flutter/material.dart';
 
+/// Returns wellbeing [charts.BarChart[ graph sent by a friend
```
Comment (thevirtuoso1973) [inline lib/shared/friend_graph.dart:6]: fix brackets

**Review by thevirtuoso1973 — COMMENTED** [review]
Shouldn't really rename the file without renaming the widget. So you can either switch back to the previous name or change all notable occurrences of 'sharing page' in the code itself.

**Comment by thevirtuoso1973** [comment]
Also fix conflicts.

File: lib/shared/wellbeing_graph.dart [hunk 2]
```diff
@@ -14,9 +14,10 @@ const RECOMMENDED_STEPS_IN_WEEK = 70000;
 /// key to retreive [bool] from [SharedPreferences] that is true if the tutorial has been completed
 const WB_TUTORIAL_DONE_KEY = "wb_tutorial_done";
 
+
 /// a [StatefulWidget] that displays the last wellbeing items in a graph,
 /// along with (optionally) a share button, and (optionally) a tutorial button
-///
+
 /// REVIEW: maybe switch to this if have time: https://pub.dev/packages/fl_chart
```
Comment (thevirtuoso1973) [inline lib/shared/wellbeing_graph.dart:21]: I think this needs to be connected for it to be considered a doc comment for WellbeingGraph widget. e.g. hover over WellbeingGraph in your editor and see if the entire comment is displayed, not just the REVIEW part.

**Review by thevirtuoso1973 — CHANGES_REQUESTED** [review]

File: lib/shared/wellbeing_graph.dart (hunk 2 above)
Comment (saachipahwa) [reply lib/shared/wellbeing_graph.dart:21]: image

Comment (saachipahwa) [reply lib/shared/wellbeing_graph.dart]: I'll connect it so it looks nicer

**Review by thevirtuoso1973 — APPROVED** [review]

**Pull request merged**